Dockerfile

```
@@ -1,3 +1,11 @@
+FROM composer AS require-server
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)